assets/schema_database.json

```diff
@@ -57,14 +57,19 @@
                 "errorMessage": "Invalid database db_params entry. No quotes allowed.",
                 "meta": ["db_params"]
             },
+            "db_type": {
+                "type": "string",
+                "enum": ["short", "long", "both"],
+                "meta": ["db_type"]
+            },
             "db_path": {
                 "type": "string",
                 "exists": true,
                 "format": "path",
                 "errorMessage": "db_path should be either a file path or a directory."
             }
         },
-        "required": ["tool", "db_name", "db_path"],
+        "required": ["tool", "db_name", "db_type", "db_path"],
```

Comment:
I sort of wonder if we should make `short` the default (or both?), and then `db_type` is then optional? This would then make the pipeline backwards compatible with older versions of the database sheet?

Reply:
That's the very good suggestion. I will fix that.

Reply:
I have set the default to "short, long" (both).